### DISCUSSION TITLE
Move the Ionic/Angular build result to its final destination folder on CentOS self-hosted server (not cloud)

### DISCUSSION THREAD

**Discussion by TI-ILDV** [community/community#38183] [Actions]
I want to know the best way to built my Ionic/Angular project and move the resulting files at the www default folder to my current configured folder for this application.

I have started my workflow file from the default Node.js workflow file. I have made it to do the build of the project, but when trying to delete alla the files at the destination folder the command won't run. The command i'm trying is `find "/destination/folder/" | rm -rf`. When i run this command at the terminal, it works just fine.

I also tried another approach going to the destination folder and deleting all files in it. Works at the terminal, but the command `cd` doesn't work in a GitHub Action. What i'm trying is 
`cd "/destination/folder/"`
`rm -rf *`

I'm new to GitHub Actions, and think there is something important i'm not understanding. Can you help me get in the correct way?

**Comment by ossanna16** [comment]
Hi there @TI-ILDV and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
